cmd/buildkitd/config/config.go

```diff
@@ -71,6 +71,8 @@ type NetworkConfig struct {
 	CNIConfigPath string `toml:"cniConfigPath"`
 	CNIBinaryPath string `toml:"cniBinaryPath"`
 	CNIPoolSize   int    `toml:"cniPoolSize"`
+	BridgeName    string `toml:"bridgeName"`
+	// BridgeSubnet  string `toml:"bridgeSubnet"`
```

Comment:
@AkihiroSuda @jedevc If we add https://github.com/moby/buildkit/issues/4099 then I think we need to add some reserved IP ranges that the tunnels created via LLB should use. If this `BridgeSubnet` collides with such ranges we could give an error and `buildkitd` would not start. Otherwise, if user would use an IP that collides with the configured bridge then it would not be possible to create such tunnels. Does this make sense?